ui/page/transaction/transactions_page.go

```diff
@@ -384,15 +427,29 @@ func (pg *TransactionsPage) HandleUserInteractions() {
 		break
 	}
 
+	if pg.isHomepageLayout && pg.walletDropDown.Changed() {
+		pg.selectedWallet = pg.assetWallets[pg.walletDropDown.SelectedIndex()]
+		pg.refreshAvailableTxType()
+		go pg.scroll.FetchScrollData(false, pg.ParentWindow())
+	}
+
 	if clicked, selectedItem := pg.transactionList.ItemClicked(); clicked {
 		transactions := pg.scroll.FetchedData().([]sharedW.Transaction)
 		pg.ParentNavigator().Display(NewTransactionDetailsPage(pg.Load, &transactions[selectedItem], false))
 	}
-	cryptomaterial.DisplayOneDropdown(pg.txTypeDropDown)
 
-	if tabItemClicked, clickedTabIndex := pg.tabs.ItemClicked(); tabItemClicked {
-		pg.selectedTabIndex = clickedTabIndex
+	dropDownList := []*cryptomaterial.DropDown{pg.txTypeDropDown}
+	if pg.isHomepageLayout {
+		dropDownList = append(dropDownList, pg.walletDropDown)
+	}
+	cryptomaterial.DisplayOneDropdown(dropDownList...)
```

Comment:
Does this actually display a dropdown???

Reply:
Displays one dropdown at a time. So its a method the minimizes an opened dropdown and open the new one.